foundry.md

```diff
@@ -116,7 +116,7 @@ This rule then takes the account using `#asWord(#range(LM, ARGSTART +Int 4, 32)`
          <output> _ => .ByteArray </output>
          <localMem> LM </localMem>
       requires CHEAT_ADDR ==Int #address(FoundryCheat)
-       andBool #asWord(#range(LM, ARGSTART, 4)) ==Int 3364511341 // selector ( "deal" )
+       andBool #asWord(#range(LM, ARGSTART, 4)) ==Int 3364511341 // selector ( "deal(address,uint256)" )
       [priority(40)]
 
```

Comment:
Can we also reformat `call.deal` rule to have `<k>` cell on one line too?

Reply:
Imo, as a one-liner it has decreased readability